spec/features/system/admin/user_management_spec.rb

```diff
@@ -41,7 +41,7 @@
         expect(user_to_change.reload).to be_administrator
       end
 
-      let!(:user_to_delete) { create(:user, name: users.first.name) }
+      let!(:user_to_delete) { create(:user, name: 'A' + users.first.name) }
```

Comment:
Use `'!'` instead of `'A'` ?  That's the smallest valid character in username I think.